Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -391,7 +392,7 @@ extension SPNoteEditorViewController {
     func presentInformationController(for note: Note, from barButtonItem: UIBarButtonItem) {
         let informationViewController = NoteInformationViewController(note: note)
 
-        let presentAsPopover = UIDevice.sp_isPad() && traitCollection.horizontalSizeClass == .regular
+        let presentAsPopover = UIDevice.isPad && traitCollection.horizontalSizeClass == .regular
```

Comment:
When building Simperium via CocaoPods, all headers are accessible. When building with SwiftPM, only those explicitly configured as public in the Xcode project and imported in the umbrella header are. 

This resulted in `sp_isPad()` no longer being available on my first Simperium-via-SwiftPM go. I could have made it public, but I noticed we already have an implementation in the app code base, so I just used that.